### ISSUE TITLE
Overlapping Product Images On Grid View On Mobile In Landscape View Using Responsive Classic Template

### ISSUE THREAD

**Issue by andy-1977** [zencart/zencart#6253]
When using Zencart v1.5.8a with the Responsive Classic Template. If Configuration > Product List > Columns Per Row (is set to 3), there is a minor bug causing rows of 2 products to show if your on a mobile phone, and have rotated your phone to the side, and are viewing in landscape view. In portrait view on a mobile, all is good, and rows of 1 product show, which is ideally what you want to see. Desktop and ipads seem ok.

The bug with 2 products showing in landscape view on mobiles, causes the product images to overlap the product descriptions, and looks out of sync.

Desktop works well, with rows of 3. 

This bug only appears to happen, when using a mobile phone in landscape view.

An ideal fix for this, would be to make sure that only rows of 1 product appear, when viewing in landscape view on mobile devices.

Zen Cart version: 1.5.8a
PHP version: 8.2

**Comment by drbyte** [comment]
I imagine this is what you're referring to:
![iPhone XR-1708893062671](https://github.com/zencart/zencart/assets/404472/b3959d6c-324f-45be-970f-558acd53d669)

Editing the `responsive_mobile.css` file, line 128, and changing `height:100%;` to `height:auto;` causes the display to change to the following:

```diff
-.list-image{margin:20px 20px 10px auto; text-align:center;display:inline-block;height:100%;vertical-align:top;}
+.list-image{margin:20px 20px 10px auto; text-align:center;display:inline-block;height:auto;vertical-align:top;}
```
![iPhone XR-1708893130714](https://github.com/zencart/zencart/assets/404472/42d9df5f-f725-4f2b-9057-54c9c6685477)

**Comment by andy-1977** [comment]
Perfect!

That works like a charm!

Thanks for such a quick fix for this issue (much appreciated!)

Thanks drbyte.

**Comment by drbyte** [comment]
> Perfect!
> That works like a charm!

Excellent.

Also note that this is fixed in v2.0.0

**Comment by andy-1977** [comment]
Many thanks for the update! 👍
